File: lobwebapp-server/src/main/java/com/espindola/lobwebapp/domain/Payment.java
package com.espindola.lobwebapp.domain;

import java.math.BigDecimal;

import javax.persistence.Entity;
import javax.persistence.Table;

import com.espindola.lobwebapp.domain.base.AbstractEntity;

@Entity
@Table(name = "TB_PAYMENT")
public class Payment extends AbstractEntity {

	private BigDecimal quantity;

	private PaymentStatus status;

	private PaymentMode mode;

	public BigDecimal getQuantity() {
		return quantity;
	}

	public void setQuantity(BigDecimal quantity) {
		this.quantity = quantity;
	}

	public PaymentStatus getStatus() {
		return status;
	}

	public void setStatus(PaymentStatus status) {
		this.status = status;
	}

	public PaymentMode getMode() {
		return mode;
	}

	public void setMode(PaymentMode mode) {
		this.mode = mode;
	}

}

File: lobwebapp-client/src/app/order/Payment.ts
///<reference path="../reference.ts"/>

module order {
  export interface Payment extends core.AbstractEntity {
    quantity: number;
    status: string;
    mode: string;
  }

  export class PaymentStatus {
    public static OK = "OK";
    public static PENDING = "PENDING";
    public static CANCELLED = "CANCELLED";
  }

  export class PaymentMode {
    public static MONEY = "MONEY";
    public static CREDIT_CARD = "CREDIT_CARD";
    public static CHECK = "CHECK";
  }

}
